src/calamares/CalamaresWindow.cpp

```diff
@@ -161,6 +161,31 @@ getWidgetNavigation( Calamares::DebugWindowManager*,
     QBoxLayout* bottomLayout = new QHBoxLayout;
     bottomLayout->addStretch();
 
+    QObject::connect( viewManager,
+                      &Calamares::ViewManager::navigationPanelWidgetChanged,
+                      bottomLayout,
+                      [ bottomLayout ]( QWidget* wgt )
+                      {
+                          if ( !bottomLayout )
```

Comment:
I'm not sure what this `if()` is supposed to protect against: it's a copy of the layout, which is also the object to which the signal is sent .. how is this ever going to be *nullptr* without lots of other UB happening already?